src/main/java/frc/robot/subsystems/TurretSubsystem.java

```diff
@@ -135,14 +144,18 @@ public Command stowTurret() {
   }
 
   private void setTurretPos(double setpoint) {
-    turretGoalPositionEntry.log(setpoint);
-    turretMotor.set(
-        -turretPIDController.calculate(
-            turretPosition,
-            MathUtil.clamp(
-                setpoint,
-                Math.toRadians(-Constants.TurretConstants.MAX_TURRET_ANGLE_DEGREES),
-                Math.toRadians(Constants.TurretConstants.MAX_TURRET_ANGLE_DEGREES))));
+    if (topLimitSwitch.get() || bottomLimitSwitch.get()) {
+      turretMotor.set(0);
```

Comment:
This will permanently stop the turret once a limit switch is hit. Really we want to prevent it from going further in the same direction.

Reply:
doesn't actually it will back off the limit switch in the periodic. So dependent on execution order